packages/compass-crud/src/components/insert-json-document.tsx

```diff
@@ -43,16 +67,22 @@ class InsertJsonDocument extends Component<InsertJsonDocumentProps> {
   }
 
   render() {
+    const darkMode = this.props.darkMode;
     let value = this.props.jsonDoc;
 
     if (this.props.isCommentNeeded) {
       value = `${EDITOR_COMMENT}${this.props.jsonDoc}`;
     }
 
     return (
-      <div className={styles['editor-container']}>
+      <div
+        className={cx(
+          editorContainerStyles,
+          darkMode ? editorContainerStylesDark : editorContainerStylesLight
+        )}
+      >
         <Editor
-          className={styles.editor}
+          className={editorStyles}
```

Comment:
Would it work if the background color is something the editor controls? I'm wondering if there are other spots that might benefit if the background color setting happens in the `Editor` component instead of here at the consumer. Fine as is too.

Reply:
I don't know. The colours are part of editor container which is outside editor. But this is not the only editor container, so if we change something there I'd rather do that in a separate PR so we can change validation and whatever else is doing that as well.

The `editorStyles` variable you commented on only sets the height.